src/configs/sort.ts

```diff
@@ -89,6 +89,93 @@ export async function sortPackageJson(): Promise<FlatConfigItem[]> {
             ],
             pathPattern: '^exports.*$',
           },
+          {
+            order: [
+              'prepare',
+              'prepack',
+              'pack',
+              'postpack',
+              'prerelease',
+              'release',
+              'postrelease',
+              'prepublish',
+              'prepublishOnly',
+              'publish',
+              'postpublish',
+              'preinstall',
+              'install',
+              'postinstall',
+              'preuninstall',
+              'uninstall',
+              'postuninstall',
+              'preversion',
+              'version',
+              'postversion',
+              { keyPattern: '^start' },
+              { keyPattern: '^restart' },
+              { keyPattern: '^stop' },
+              'dev',
+              { keyPattern: '^build' },
+              'serve',
+              'preview',
+              'typecheck',
+              'type-check',
+              { keyPattern: '^lint' },
+              { keyPattern: '^test' },
+            ],
+            pathPattern: '^scripts$',
```

Comment:
It's deliberately not included, as there are different opinions/conventions to have the scripts. It might be better to leave the sorting to the users.